src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx

```diff
@@ -191,23 +195,24 @@ const Profile = () => {
                             <Stack>
                               <Typography variant="h6">{username}</Typography>
                               <Typography variant="body2" color="textSecondary">
-                                Dude
+                                {role === 'admin' ? 'admin' : 'volunteer'}
```

Comment:
```suggestion
                               {role}
```
Can we do this? In case there are other roles in the future, we get the new role. And if it is blank, at least we don't see volunteer.

Reply:
Yes I've updated it.